src/backend/marsha/core/utils/api_utils.py

```diff
@@ -49,3 +50,8 @@ def generate_hash(secret, message):
     return hmac.new(
         secret.encode("utf-8"), msg=message, digestmod=hashlib.sha256
     ).hexdigest()
+
+
+def generate_salted_hmac(secret, key):
+    """Generate a salted_hmac with secret and key"""
+    return salted_hmac(secret, key, hashlib.sha256).hexdigest()[::2]
```

Comment:
And why using `[::2]` ?

Reply:
to make it smaller, I'm an old dev lol and URL used to be limited to 255 chars :-) there's no need for it to be that long, I find it scary

Reply:
Oh no don't worry, the limit is much more higher, don't be shy https://stackoverflow.com/a/417184